Update redirected URLs (part 33) (#15582)
mdn/content: files/en-us/learn/tools_and_testing/cross_browser_testing/your_own_automation_environment/index.md

If you click on the link for your test, you'll get to a new screen where you will be able to see a video recording of the test, and multiple detailed logs of information pertaining to it.

> **Note:** The _Resources_ menu option on the Browserstack automation dashboard contains a wealth of useful information on using it to run automated tests. See [Node JS Documentation for writing automate test scripts in Node JS](https://www.browserstack.com/docs/automate/selenium/getting-started/nodejs) for the node-specific information. Explore the docs to find out all the useful things BrowserStack can do.

> > **Note:** If you don't want to write out the capabilities objects for your tests by hand, you can generate them using the generators embedded in the docs. See [Run tests on mobile browsers](https://www.browserstack.com/docs/automate/selenium/getting-started/nodejs#run-tests-on-mobile) and [Run tests on desktop browsers](https://www.browserstack.com/docs/automate/selenium/getting-started/nodejs#setting-os-and-browser).

Author: ```suggestion
> **Note:** If you don't want to write out the capabilities objects for your tests by hand, you can generate them using the generators embedded in the docs. See [Run your first test](https://www.browserstack.com/docs/automate/selenium/getting-started/nodejs#run-your-first-test).
```
